scm/src/multi_run_scm.py

```diff
@@ -6,13 +6,17 @@
 import os
 import logging
 import subprocess
-from supported_suites import suites
+from suite_info import suite, suite_list
 from supported_cases import cases
 import timeit, functools
 
 # Name of the python runscript executable to run, including path (relative to run dir)
 RUN_SCRIPT = './run_scm.py'
 
+# which suites in suite_info to use when running through suites
+SUITE_CHOICE = 'supported'
```

Comment:
This isn't necessarily something that I expect users to use. I want to keep the functionality such that when one specifies a case, that only the supported suites are run. I added this in case we, as the developers, ever wanted to run through all suites instead of only supported ones, e.g. for regression tests. I suppose if that time comes, then we'll need to add it as an optional command line argument with the default as 'supported' so that we can call the script from the rt.sh script and control this. For now, though, it operates exactly as described in the docs.